reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -7,28 +7,26 @@ title:  about_Remote_Variables
 ---
 # About Remote Variables
 
-## SHORT DESCRIPTION
-
+## Short Description
 Explains how to use local and remote variables in remote commands.
 
-## LONG DESCRIPTION
+## Long Description
 
 You can use variables in commands that you run on remote computers. Simply
-assign a value to the variable and then use the variable in place of the
-value.
+assign a value to the variable and then use the variable in place of the value.
 
 By default, the variables in remote commands are assumed to be defined in the
 session in which the command runs. You can also use variables that are defined
 in the local session, but you must identify them as local variables in the
 command.
 
-## USING REMOTE VARIABLES
+## Using remote variables
 
-Windows PowerShell assumes that the variables used in remote commands are
-defined in the session in which the command runs.
+PowerShell assumes that the variables used in remote commands are defined in
+the session in which the command runs.
 
-In the following example, the \$ps variable is defined in the temporary session
-in which the Get-WinEvent command runs.
+In the following example, the `$ps` variable is defined in the temporary
+session in which the Get-WinEvent command runs.
```

Comment:
Should codefence Get-WinEvent for consistency.

Reply:
Fixed